src/main/scala/ai/datahunters/md/config/enrich/MetadataEnrichmentConfig.scala

```diff
@@ -0,0 +1,42 @@
+package ai.datahunters.md.config.enrich
+
+import ai.datahunters.md.config.ConfigLoader.assignDefaults
+import ai.datahunters.md.config.ConfigLoader.ListElementsDelimiter
+import com.typesafe.config.Config
+
+case class MetadataEnrichmentConfig(labelsMapping: Map[Int, String],
+                                    threshold: Float,
+                                    modelPath: String,
+                                    labelsOutputDelimiter: String)
```

Comment:
Not so important, but I would suggest to change name from labelsOutputDelimiter to outputLabelsDelimiter. 
This way same naming convention will be covered( with val OutputLabelsDelimiterKey).

Reply:
@dawidsau It's not so obvious ;) because here we have delimiter for output of labels. Different thing is labels output :P but outputLabelsDelimiter sounds better and wil be valid here as well ;)